lib/formatters/html.js

```diff
@@ -74,6 +75,13 @@ function renderMessages(messages, parentIndex) {
     return lodash.map(messages, message => {
         const lineNumber = message.line || 0;
         const columnNumber = message.column || 0;
+        let ruleUrl;
+
+        if (rulesMeta) {
```

Comment:
Could `rulesMeta` be passed as an argument to this function rather than setting a global?